src/game-container/game-gfx-container/GfxContainer.svelte

```diff
@@ -1,56 +1,161 @@
 <script>
-    import utilFunctions from '../../helpers/utilFunctions'
-    import GameBlockRow from './GameBlockRow.svelte'
+    import util from '../../helpers/utilFunctions'
+    import initOverworld from '../../game/initOverworld'
+    import createCharInstance from '../../game/createCharInstance'
 
-    console.log(utilFunctions)
+    export let classList
 
-    utilFunctions.docReady(function() {
+    let gameChar = {}
+    let frontCtx;
+    let backCtx;
 
-        const gridWidth     = document.getElementById("game-gfx-body").offsetWidth
-        const gridHeight    = document.getElementById("game-gfx-body").offsetHeight
-        const gridDimension = [ 48, 32 ]
-        const blockDimensions = gridWidth / 48
+    function startGame() {
+        const charName      =  util.getInputVal('name')
+        const charGender    =  util.getInputVal('gender')
+        const charClass     =  util.getInputVal('class')
 
-        console.log(gridWidth / 24)
-        console.log(gridHeight / 16)
+        // The setTimeouts setup is not definitive and might change later
+        setTimeout( () => {
+            document.getElementById('intro-screen').remove()
+
+        }, 50 )
+
+        setTimeout( () => {
+            backCtx     = initOverworld.initCanvas(0)      
+            frontCtx    = initOverworld.initCanvas(1)     
+        }, 75 )
+
+        setTimeout( () => {
+            const playerCharacter = createCharInstance.getCharacter( charClass, charName, charClass )           
+
+            gameChar = playerCharacter
+
+        }, 100 )
         
+    }
+	
+    function handleInput ( event ) {
+        switch (event.key) {
+            // fallthrough for movement
+            // this switch statement and associated functionalities
+            // are still in an experimental phase
+            case 's' :
+            case 'a' :
+            case 'w' : 
+            case 'd' :
+            case 'ArrowRight' :
+            case 'ArrowUp' :
+            case 'ArrowLeft' :
+            case 'ArrowDown' :
+                handleMovement(event.key)
+            case 'Tab' :
+            case 'i' :
+            case 'o' :
+                /* openMenu(event.key) */
+            case ' ' :
+                /* handleActionButton() */
+        }
+    }
 
-    });
+    util.docReady(
+        window.addEventListener('keydown', handleInput)
+    )
 
-    const flex_wrap = 100 / 32
+    function handleMovement(key) {
```

Comment:
Je doet niks met de key? Ik zie niet helemaal in hoe je character hier gaat lopen

Reply:
Ja dit is nu al helemaal veranderd in een andere branch, maar thx voor de heads-up